### PR TITLE
SSCS-10803 allow clerk and judge to action postponement

### PR DESCRIPTION
**Pull request by pk2453**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```
https://tools.hmcts.net/jira/browse/SSCS-10803
https://tools.hmcts.net/jira/browse/SSCS-10789

### CODE REVIEW

**Comment by pk2453** [comment]
version bump will be done once approved

**Review by ikirsanov — APPROVED** [review]

**Review by nickhill111 — DISMISSED** [review]

**Review by dattipoeHMCTS — APPROVED** [review]

File: benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json [hunk 1]
```diff
@@ -1201,6 +1201,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "unprocessedPostponementRequest", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-judge", "CRUD": "CRUD"},
```
Comment (dattipoeHMCTS) [inline benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json:1204]: All these changes should be in the non prod sheets

**Review by dattipoeHMCTS — CHANGES_REQUESTED** [review]

File: benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json (hunk 1 above)
Comment (bensonlievoco) [reply benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json:1204]: it is fine

Comment (dattipoeHMCTS) [reply benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json:1204]: It does need to be in the non prod sheet. @filippo-possenti-version1

Comment (filippo-possenti-version1) [reply benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json]: Done.

**Review by dattipoeHMCTS — APPROVED** [review]

**Review by bensonlievoco — APPROVED** [review]

**Pull request merged**